$NetBSD$

Increase limit for open files per process to hard limits of system.

--- scripts/rc.news.in.orig	2019-02-07 15:02:22.000000000 +0000
+++ scripts/rc.news.in
@@ -77,6 +77,9 @@ Xstop)
 ;;
 esac
 
+## Unlimit
+ulimit -n `ulimit -H -n`
+
 ##  Pick ${INND} or ${INNDSTART}
 WHAT=${INND}
 
